node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/maintenance/servicedump/VespaServiceDumperImplTest.java

```diff
@@ -73,7 +74,7 @@ void generates_jvm_dump_from_request() throws IOException {
         NodeSpec initialSpec = createNodeSpecWithDumpRequest(nodeRepository, JvmDumpProducer.NAME, null);
 
         // Create dumper and invoke tested method
-        VespaServiceDumper reporter = new VespaServiceDumperImpl(operations, syncClient, nodeRepository, clock);
+        VespaServiceDumper reporter = new VespaServiceDumperImpl(operations, syncClient, nodeRepository, clock, Sleeper.NOOP);
```

Comment:
Note that you could do `mock(Sleeper.class)` here, and let Sleeper be a class with the Sleeper.DEFAULT implementation.  This would avoid the interface, 2 constants, and 2 implementations (as top-level declarations in the Sleeper.java file, which I have not encountered elsewhere in our code).